gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=http\://services.gradle.org/distributions/gradle-4.0-bin.zip
+distributionUrl=http\://services.gradle.org/distributions/gradle-4.4.1-all.zip
```

Comment:
Please use the normal distribution. Having the fat one is a pain for lots of users like me with low bandwidth.

Reply:
fixed